docs/Automation.md

```diff
@@ -20,164 +15,103 @@ and is not expected to change at any future time. If you have a scenario that
-configFile | `string` | The full path to a file that contains JSON-serialized Tier 3 parameters. Can be `string.Empty`. Must point to a valid JSON-serialized file if non-empty.
+Consumers should look to follow the processs below:
 
-###### PowerShell
+1. Create `Config` using `ConfigBuilder`
+2. Create `Scanner` using `Config` created in step 1.
+3. Run `Scan` method on `Scanner` object.
+4. Analyze the returned `ScanResults` object for your respective needs.
 
-The **Start-AxeWindows** cmdlet accepts the following optional
-parameters:
+The details of these objects/methods are below.
 
-**Name** | **Type** | **Description**
----|---|---
-OutputFile | `string`  | The location where output from this session will be written. The file extension can (and should) be omitted. This parameter can be specified either at Start-AxeWindows or at Invoke-Snapshot.
-OutputFileFormat | `string` | The output file format to generate. This parameter can be specified either at Start-AxeWindows or at Invoke-Snapshot.
-TargetProcessId | `string` | The process id, expressed as a string, that will be scanned. This parameter can be specified either at Start-AxeWindows or at Invoke-Snapshot.
-ConfigFile | `string` | The full path to a file that contains JSON-serialized Tier 3 parameters. Must point to a valid JSON-serialized file if specified.
+### Implementation Details
 
-##### Return object
+#### Config.Builder Details/Methods
+
+##### ForProcessId
+Create the builder for the config for the specified process.
+
+##### Parameters
 
-The **StartCommandObject** has the following properties:
+The **ForProcessId** method accepts the following parameters:
 
 **Name** | **Type** | **Description**
 ---|---|---
-Completed | `bool` | True if the command ran to completion. If true, then the Succeeded property indicates if the call succeeded.
-SummaryMessage | `string` | A human-readable message to summarize the result. This message can change at any time, so do not rely on parsing it in tools.
-Succeeded | `bool` | True if the Start command completed successfully.
+processId | `int` | The process Id of the application to test. If the value is invalid, the automation session will throw an `AxeWindowsAutomationException`.
 
-#### Snapshot Command
-The Snapshot command scans a single process for accessibility issues and saves
-the output file for later analysis. It can only be called while an automation
-session is open (i.e., between the Start and Stop commands):
+##### Return object
 
-##### Inputs
+The **ForProcessId** method returns a **Config.Builder** object, which has the following methods:
 
-###### .NET
+**Name** | **Parameters** | **Output** | **Description**
+---|---|---|---
+WithOutputFileFormat | `(OutputFileFormat format)` | `Config.Builder` | Specify the type(s) of output files you wish AxeWindows to create. No output files will be created if this is left unspecified.
```

Comment:
>OutputFileFormat [](start = 25, length = 16)

Suggestion: Document this enum and make this be a link to the enum. The enum will only have 1 value for now, but we'll soon be adding a second value

Reply:
In the comments for this, it points to AIWin, should we also link to it?

Reply:
Yes